test/README.md

```diff
@@ -1,6 +1,50 @@
-This directory contains the WebAssembly test suite. It is split into two directories:
+This directory contains the WebAssembly test suite. It is split into two
+directories:
 
 * [`core/`](core/), tests for the core semantics
-* [`js-api/`](js-api/), tests for the JavaScript API
+* [`js-api/`](js-api/), tests for the JavaScript API. These tests can be run in
```

Comment:
Nit: I'd prefer not to front-load the bullet list with too much information. Maybe we should have sections on each one below.